ui/apps/platform/cypress/integration/policies/importPolicy.test.js

```diff
@@ -253,4 +253,12 @@ describe('Import policy', () => {
         cy.get(selectors.importUploadModal.renameInput).type('Two are better than one');
         cy.get(selectors.importUploadModal.resumeButton).should('be.enabled');
     });
+
+    it('should have no detectable accessibility violations in the modal', () => {
+        visitPolicies();
+
+        cy.get(selectors.table.importButton).click();
+
+        cy.checkAccessibility(selectors.importUploadModal.modalWrapper);
```

Comment:
A question but not a request for change. Is there a benefit to including context here? It seems to me that including the full page context would make us more likely to catch a11y regressions, even if they were technically outside of the exact scope being tested.

One use case I can think of for this is where we want to `checkAccessibility` on a context where we know there are existing errors on the page, but are unable to address the errors elsewhere as part of the current change.

Reply:
Insightful question. The second paragraph is the only use case that comes to mind.
* Browser plugin required paid account to check limited page context, so default to entire page in integration tests is consistent with manual testing.
* In the specific example of modal, an accessibility regression in the product came to our attention while investigating an accessibility error in a innocent modal.

Reply:
Yeah that is a great question. My thinking was similar to what you mentioned in your second paragraph. Although I can see the benefit of scanning the full page here since there could be accessibility issues in the surrounding DOM that were introduced by the modal, but are outside of the targeted context..

I am imagining full context will make sense in most instances, but there are situations where it might be better to focus on a specific section to avoid being distracted by unrelated issues, especially since the tests fail as soon as the first issue is found. For example, with a more complex modal where we need to run multiple accessibility checks while modifying its state, I wouldn’t want the test to fail due to an issue in the background that would already be caught by a test in `policiesTable.test.ts`. However, this approach would require us to be diligent in ensuring we’re also testing accessibility outside of the context.